src/support/mtx_rw.c

```diff
@@ -316,31 +330,32 @@ __wt_writelock(WT_SESSION_IMPL *session, WT_RWLOCK *rwlock)
 int
 __wt_writeunlock(WT_SESSION_IMPL *session, WT_RWLOCK *rwlock)
 {
-	wt_rwlock_t *l, copy;
+	wt_rwlock_t *l, new;
```

Comment:
This variable rename is just for consistency with other functions in the same file.